app/Resources/views/admin/layout.html.twig

```diff
@@ -7,6 +7,9 @@
 #}
 {% extends 'base.html.twig' %}
 
+{# Import macros needed for displaying flash messages. See http://twig.sensiolabs.org/doc/tags/import.html #}
+{% import 'macro/messages.html.twig' as messages %}
```

Comment:
macros should be imported in the template where they are used. Importing them in a parent template and expecting them to work in child templates is just a side effect of the Twig 1.x implementation, and does not work anymore in Twig 2